.gitignore

```diff
@@ -45,3 +45,6 @@ msbuild.log
 msbuild.err
 msbuild.wrn
 OpenAPI
+
+appsettings.json
+appsettings.*.json
```

Comment:
This should probably go to the project's gitignore instead of the global one. Otherwise it affects files that are already committed to the repo.

Reply:
Done ✅